.spelling

```diff
@@ -316,6 +316,7 @@ HTTP1.1
 HTTP2
 http2
 httpbin
+httpbin.foo.svc.cluster.local
```

Comment:
This is for using `<code>` and `&#124;` to workaround the `|` and spelling error in the mark down table in index.md.